translate/filters/test_checks.py

```diff
@@ -1272,6 +1272,155 @@ def test_nplurals():
     assert not checker.nplurals(unit)
 
 
+def test_untranslated_plurals_no_plural_unit():
+    """Test that we do not detect untranslated plurals on units with no plural.
+
+    Note that this test uses a UnitChecker, not a translation checker.
+    """
+    unit = po.pounit("")
+    unit.source = "%d files"
+
+    # No nplurals set.
+    checker = checks.StandardUnitChecker()
+
+    unit.target = ""
+    assert checker.untranslated_plurals(unit)
+    unit.target = "%d lêer"
+    assert checker.untranslated_plurals(unit)
+
+    # nplurals == 1 set.
+    checker = checks.StandardUnitChecker(
+        checks.CheckerConfig(targetlanguage='km'))
+
+    unit.target = ""
+    assert checker.untranslated_plurals(unit)
+    unit.target = "%d lêer"
+    assert checker.untranslated_plurals(unit)
+
+    # nplurals == 2 set.
+    checker = checks.StandardUnitChecker(
+        checks.CheckerConfig(targetlanguage='af'))
+
+    unit.target = ""
+    assert checker.untranslated_plurals(unit)
+    unit.target = "%d lêer"
+    assert checker.untranslated_plurals(unit)
+
+    # nplurals == 6 set.
+    checker = checks.StandardUnitChecker(
+        checks.CheckerConfig(targetlanguage='ar'))
+
+    unit.target = ""
+    assert checker.untranslated_plurals(unit)
+    unit.target = "%d lêer"
+    assert checker.untranslated_plurals(unit)
+
+
+def test_untranslated_plurals_unit_with_plurals():
+    """Test that we detect untranslated plurals.
+
+    Note that this test uses a UnitChecker, not a translation checker.
+    """
+    unit = po.pounit("")
+    unit.source = ["%d file", "%d files"]
+
+    # No nplurals set.
```

Comment:
I think the comment should state what you expect to happen.  IIUC there should be no failures.